server/datastore/mysql/queries.go

```diff
@@ -90,6 +101,18 @@ func (ds *Datastore) ApplyQueries(ctx context.Context, authorID uint, queries []
 		}
 	}
 
+	for id := range queriesToDiscardResults {
+		if resultsStmt != nil {
```

Comment:
It cannot be `nil` at this point, right?

Reply:
derp yes you're right, I copied this block into this `for` loop and forgot to remove that check.